internal/component/database_observability/mysql/collector/schema_table.go

```diff
@@ -98,13 +100,15 @@ type SchemaTable struct {
 }
 
 type tableInfo struct {
-	schema     string
-	tableName  string
-	tableType  string
-	createTime time.Time
-	updateTime time.Time
-	createStmt string
-	tableSpec  string
+	schema        string
+	tableName     string
+	tableType     string
+	createTime    time.Time
+	updateTime    time.Time
+	tableRows     int64
+	autoIncrement int64
+	b64CreateStmt string
+	b64TableSpec  string
```

Comment:
Drive-by change to store `createStmt` and `tableSpec` in base64 to avoid encoding them at log time.